frontend/src/export/mtgo.spec.js

```diff
@@ -0,0 +1,43 @@
+const {describe, it} = require("mocha");
+const assert = require("assert").strict;
+
+import { exampleDeck as deck } from "./test-cards.js";
+import mtgo from "./mtgo.js";
+
+describe("export/mtgo", () => {
+  // console.log(multiFacettedCards.map(c => ([c.name, c.layout])))
+  it("download of .dek file", () => {
+    assert.equal(
+      mtgo.download("My deck", deck),
+      downloadOutput(),
+      "correct output"
+    );
+  });
+
+  it("copy to clipboard", () => {
+    assert.equal(mtgo.copy, undefined, "copy to clipboard disabled");
+
+    // assert.equal(
+    //   mtgo.copy("My deck", deck),
+    //   copyOutput(),
+    //   "correct output"
```

Comment:
Same question, should it be deleted or kept ?

Reply:
Those is again hinting about what code should be there to help a future person adding a test. It helps it fit the pattern with other exports.

Happy to delete if it's not a useful breadcrumb

Reply:
removed